common/util_test.go

```diff
@@ -120,8 +120,20 @@ func TestCreateHistoryStartWorkflowRequest_DelayStart(t *testing.T) {
 
 	expirationTime := startRequest.GetExpirationTimestamp()
 	require.NotNil(t, expirationTime)
-	require.True(t, time.Unix(0, expirationTime).Sub(now) > (100+60)*time.Second)
-	require.True(t, time.Unix(0, expirationTime).Sub(now) < (100+65)*time.Second)
+	require.True(
```

Comment:
Add the same explanation as comments so other people understand what's going on here?